src/pages/ProductDetail/ProductDetail.js

```diff
@@ -0,0 +1,189 @@
+import React from 'react';
+import '../ProductDetail/ProductDetail.scss';
+
+class ProductDetail extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      imgList: [
+        {
+          src: '/images/ProductDetail/mockProduct.png',
+        },
+        {
+          src: '/images/ProductDetail/mockProduct.jpg',
+        },
+        {
+          src: '/images/ProductDetail/mockProduct.jpg',
+        },
+        {
+          src: '/images/ProductDetail/mockProduct.jpg',
+        },
+        {
+          src: '/images/ProductDetail/mockProduct.jpg',
+        },
+      ],
+    };
+  }
+
+  imgChange = e => {
+    console.log(e.target);
+  };
+
+  render() {
+    console.log(this.state.imgList);
+    return (
+      <article className="productDetail">
+        <div className="container">
+          <section className="ProductDetailTop">
+            <article className="imageBox">
+              <div className="mainImage">
+                <img src="/images/ProductDetail/mockProduct.png" alt="" />
+              </div>
+              <div className="imageExpandBtn">
+                <i class="fas fa-expand"></i>
+                <span>EXPAND VIEW</span>
+              </div>
+              <div className="imageList">
+                {this.state.imgList.map((item, index) => {
+                  return (
+                    <div>
+                      <img
+                        src={item.src}
+                        alt="두번째 이미지"
+                        key={index}
+                        onClick={this.imgChange}
+                      />
+                    </div>
```

Comment:
- key props 는 가장 바깥 요소에 부여해주셔야 합니다!
```suggestion
                    <div key={index}>
                      <img
                        src={item.src}
                        alt="두번째 이미지"
                        onClick={this.imgChange}
                      />
                    </div>
```